inscriptions/ISic001137.xml

```diff
@@ -190,29 +195,19 @@
     </facsimile>
     <text>
         <body>
-            <div type="edition" xml:space="preserve" subtype="PHI" n="PHI140622">
+            <div type="edition" xml:space="preserve" xml:lang="grc" resp="#MH">
                 <ab>
-                    <lb n="1"/>
-
-
-                    Βαλ
-                    <supplied reason="lost">ερ</supplied>
-                    <supplied reason="omitted">ί</supplied>
-                    <expan>
-                        <abbr>α</abbr>
-                        <ex>?</ex>
-                    </expan>
-                    <lb n="2"/>ἡ τὸν βίο<supplied reason="lost">ν εὐ</supplied>-<lb n="3"/>πρεπ<supplied reason="omitted">ῶ</supplied>ς β<supplied reason="lost">ιώσα</supplied>-<lb n="4"/>σα <unclear>
-                    ἔ</unclear>ζη
-                    <supplied reason="lost">σεν ἔτη —ʹ</supplied>
-                    <lb n="5"/>
+                    <lb n="1"/><name>Βαλ<supplied reason="lost">ερ</supplied><choice><corr>ί</corr><sic>k</sic></choice>α</name> <gap reason="lost" extent="unknown" unit="character"/>
+                    <lb n="2"/>ἡ τὸν βίο<supplied reason="lost">ν εὐ</supplied>
+                    <lb n="3" break="no"/>πρεπ<choice><corr>ῶ</corr><sic>ο</sic></choice> <choice><corr>β</corr><sic>ρ</sic></choice><supplied reason="lost">ιώσα</supplied>
+                    <lb n="4" break="no"/>σα <choice><corr>ἔ</corr><sic>σ</sic></choice>ζη<supplied reason="lost">σεν ἔτη <desc>number</desc></supplied>
                 </ab>
             </div>
             <div type="apparatus">
                 <p><!--external apparatus criticus (if applicable)--></p>
```

Comment:
Amend the apparatus in all cases, firstly by deleting the existing `<p>` element, and then by adding:
`<listApp><app><note>Text according to Kaibel in IG</note></app></listApp>` (or equivalent)
but then, where as here you jhave alternatives to report, e.g.
`<app loc="line 1"><note>Gualterus: ΒΑΛ.ΚΑ..</note></app> `(all obviously inside the main `<listApp>`)